tests/AppBundle/EdgeToEdge/UserLoginTest.php

```diff
@@ -69,4 +73,76 @@ public function testUserLoginFrontend()
         $this->isSuccessful($client->getResponse());
         $this->assertTrue($crawler->filter('Anmelden')->count() === 0, 'Button "Anmelden" found');
     }
+
+    /**
+     * @dataProvider urlProvider
```

Comment:
What would be correct in case of a dataprovider:
```
/**
* @param array[] $url An array of arrays containing a string.
* @dataProvider urlProvider
*/
```
more clear but not exact would be
`*@param string[]` or `*@param mixed[]`